backend/src/models/user/postgres.js

```diff
@@ -96,11 +98,18 @@ const UserRepo = (postgres) => {
    */
   const createUser = async (first_name, last_name, email, passhash, pic_url, address_1, address_2, city, state, zip, phone, token) => {
     const values = [first_name, last_name, email, passhash, pic_url, address_1, address_2, city, state, zip, phone, token];
+	const emails = [email];
     try {
       const client = await postgres.connect();
+	  const check = await client.query(checkUserSQL, emails); 
+	  if(check.rows.length == 0){
```

Comment:
I'm not sure you need this, the database should already throw an error since the email column has a unique key. The error message printed out by the backend in the console is ugly but it's still a 400 and not a crash. But we could keep idk

Reply:
Also try to keep spacing/tabs consistent

Reply:
I really didn't like the error given by backend. So I added this while changing item. I can revert the changes if you want